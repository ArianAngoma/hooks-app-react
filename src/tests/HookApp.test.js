import {shallow} from "enzyme";
import HooksApp from "../HooksApp";

describe('Test to component <HookApp/>', () => {
    let wrapper;
    beforeEach(() => {
        wrapper = shallow(<HooksApp/>);
    });

    test('Should display the component <HooApp/> correctly', () => {
        expect(wrapper).toMatchSnapshot();
    });
})