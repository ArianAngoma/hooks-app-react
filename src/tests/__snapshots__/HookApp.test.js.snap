// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test to component <HookApp/> Should display the component <HooApp/> correctly 1`] = `
<div>
  <h1>
    Hola Mundo
  </h1>
</div>
`;
